src/NativeQueryable/EnumerableExecutor.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.

```

Comment:
All of the files in the NativeQueryable folder come directly from System.Linq.Queryable without change. I modify these files in the next PR.